code/_globalvars/traits/_traits.dm

```diff
@@ -499,6 +500,7 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_STRENGTH" = TRAIT_STRENGTH,
 		"TRAIT_STIMMED" = TRAIT_STIMMED,
 		"TRAIT_BOXING_READY" = TRAIT_BOXING_READY,
+		"TRAIT_MANSUS_TOUCHED" = TRAIT_MANSUS_TOUCHED,
```

Comment:
ABC order

Reply:
boxign right above dosent onform to it >:(

Reply:
it's bad too